packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -461,7 +461,7 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable, M
         if (cachedCurrencies[currency].isWhitelisted) return true;
         cachedCurrencies[currency].isWhitelisted = _getCollateralWhitelist().isOnWhitelist(currency);
         cachedCurrencies[currency].finalFee = _getStore().computeFinalFee(currency).rawValue;
-        return cachedCurrencies[currency].isWhitelisted;
+        return cachedCurrencies[currency].isWhitelisted && cachedCurrencies[currency].finalFee > 0;
```

Comment:
I think there's still an edge case where syncUmaParams could be called to cache the currency with 0 final fee and then this method would return on line 461.

Thoughts on making 461:
```
if (cachedCurrencies[currency].isWhitelisted && cachedCurrencies[currency].finalFee > 0) return true;
```

Reply:
right. also we are not sure about making this change just yet as it makes things a bit more difficult on testnets.